benchmarks/benchmarkInnerProductKernels.cpp

```diff
@@ -24,77 +24,109 @@ namespace LvArray
 namespace benchmarking
 {
 
-#define INNER_PRODUCT_KERNEL( a_i, b_i ) \
+#define INNER_PRODUCT_KERNEL( N, a_i, b_i ) \
   VALUE_TYPE sum = 0; \
   for( INDEX_TYPE i = 0; i < N; ++i ) \
   { \
     sum += a_i * b_i; \
```

Comment:
```suggestion
   sum = sum + a_i * b_i; \
```

will result in an FMA. Or take a look in nsightCompute to check if you are getting it with this....but I didn't get it in the FEM kernel.

Reply:
Done.